app/frontend/src/features/NotFoundPage.tsx

```diff
@@ -1,10 +1,12 @@
-import { Box } from "@material-ui/core";
+import { Box, Link as MuiLink } from "@material-ui/core";
 import TextBody from "components/TextBody";
-import React from "react";
 import { Link } from "react-router-dom";
 import Graphic from "resources/404graphic.png";
 import makeStyles from "utils/makeStyles";
 
+import { baseRoute } from "routes";
+import { DO_YOU_WANT, GO_HOME, NOT_FOUND } from "./constants";
```

Comment:
This looks like import order linting would fail. Did you also run `yarn lint --fix`? 
Sorry, I should have mentioned that too - but do go through the frontend checklist!